### PR TITLE
ci: build pypy wheels for macOS and Linux

### PR DESCRIPTION
**Pull request by messense**

### CODE REVIEW

File: .github/workflows/python.yaml [hunk 1]
```diff
@@ -227,6 +227,41 @@ jobs:
...
+          - '3.9'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+        with:
+          python-version: pypy-${{ matrix.python-version }}
+      - name: Build wheels
+        uses: messense/maturin-action@v1
+        with:
+          maturin-version: v0.13.0-beta.4
```
Comment (messense) [inline .github/workflows/python.yaml:249]: Use 0.13 beta for some bugfixes related to pypy and abi3 feature.

**Review by adriangb — APPROVED** [review]

**Comment by adriangb** [comment]
Awesome, thank you @messense! Let me know when you're ready to merge.

**Comment by messense** [comment]
@adriangb I think it's ready to merge if you're ok with using maturin 0.13 beta version to build pypy wheels for now.

**Comment by messense** [comment]
FYI, maturin 0.13.0 is out.

**Pull request merged**